mlog/log.go

```diff
@@ -62,25 +62,26 @@ func getZapLevel(level string) zapcore.Level {
 	}
 }
 
+func makeEncoder(json bool) zapcore.Encoder {
+	encoderConfig := zap.NewProductionEncoderConfig()
+	if json {
+		return zapcore.NewJSONEncoder(encoderConfig)
+	}
+
+	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	return zapcore.NewConsoleEncoder(encoderConfig)
+}
+
 func NewLogger(config *LoggerConfiguration) *Logger {
 	cores := []zapcore.Core{}
 	logger := &Logger{
 		consoleLevel: zap.NewAtomicLevelAt(getZapLevel(config.ConsoleLevel)),
 		fileLevel:    zap.NewAtomicLevelAt(getZapLevel(config.FileLevel)),
 	}
 
-	encoderConfig := zap.NewProductionEncoderConfig()
-	var encoder zapcore.Encoder
-	if config.ConsoleJson {
```

Comment:
The `config.ConsoleJson` field was incorrectly managing the `encoder` used for logging to a file.